### DISCUSSION TITLE
Render textures

### DISCUSSION THREAD

**Discussion by Nv7-GitHub** [hajimehoshi/ebiten#1640] [Q&A]
Are there render textures in Ebiten? Is it possible to make post-processing shaders?

**Comment by hajimehoshi** [comment]
Hi. Do you mean an offscreen? If so, you can create an empty image by `ebiten.NewImage` and then use it as a rendering target. Then you can use it as a rendering source later.

> Is it possible to make post-processing shaders?

Yes. See [the shaders](https://ebiten.org/documents/shader.html).
